depmapomics/tracker.py

```diff
@@ -11,42 +13,139 @@
 import signal
 
 
-def getTracker():
```

Comment:
this should be its own repo as well. let's move it out of depmap_omics

Reply:
Are you saying the code for reading and writing gumbo tables should be its own repository?

Reply:
yes. I think it is something that should be worked on by Omics, Achilles, .... everyone using gumbo. These are pretty general functions

Reply:
@5im1z